src/applications/static-pages/covid-vaccine-updates-cta/widget.jsx

```diff
@@ -1,19 +1,47 @@
 import React from 'react';
 import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
 import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
-import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
 import { connect } from 'react-redux';
-
 import OnState from './On';
 import OffState from './Off';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
 
-function CovidVaccineUpdatesCTA({ showLinkToOnlineForm }) {
+// TODO: follow on PR will introduce react-18-next, this is to get the word out stat
+const copy = {
+  en: {
+    appTitle: 'Covid 19 Vaccination Information',
+    cta: `Sign up for an easy way to stay informed about getting a COVID-19
+      vaccine at VA.`,
+    body: `   When you sign up, we’ll also ask about your vaccine plans. Your
+      local VA health facility may use this information to determine
+      when to contact you once your risk group becomes eligible.`,
+    boldedNote: `Note:`,
+    note: `We’ll contact every eligible Veteran in
+      each risk group. You don't need to sign up to get a vaccine`,
+    headline: 'Stay informed about getting a COVID-19 vaccine',
+    buttonText: 'Sign up to stay informed',
+  },
+  es: {
+    appTitle: `Información de vacunación contra Covid 19`,
+    buttonText: `Regístrese para mantenerse informado (en inglés)`,
+    headline:
+      'Manténgase informado sobre cómo recibir la vacuna contra COVID-19',
+    cta:
+      'Inscríbase para una manera fácil de mantenerse informado sobre cómo recibir la vacuna contra COVID-19 en el VA.',
+    body:
+      'Cuando se inscriba, también le preguntaremos acerca de sus planes de vacunas. Su centro de salud del VA local puede usar esta información para determinar cuándo contactarle una vez que su grupo de riesgo sea elegible. Por favor, tenga presente que estas actualizaciones solo están disponibles en inglés en este momento.',
+    boldedNote: 'Nota: ',
+    note:
+      'Intentaremos contactar a todos los Veteranos elegibles en cada grupo de riesgo. No es necesario que se inscriba para recibir la vacuna.',
+  },
+};
```

Comment:
Just curious - why this content isn't in Drupal?

Reply:
Bc it is rendered inside of dynamic content - content driven off of a Flipper feature flag